### ISSUE TITLE
Still maintained?

### ISSUE THREAD

**Issue by dandv** [lyricat/Hotot#590]
Site is down.

If the project is no longer maintained, it would be great to archive the repo so that it readily appears as no longer maintained in GitHub searches.

https://help.github.com/articles/archiving-a-github-repository/

**Comment by benaryorg** [comment]
No, the project is no longer maintained, and yes, it would be great if the original maintainers did a thing and e.g. gave access to the repository to other people, but I'm afraid this will not happen anytime soon (note that the last release/commit was 2015-08-12).
